waltz-client/src/main/java/com/wepay/waltz/client/WaltzClientConfig.java

```diff
@@ -95,4 +114,18 @@ public WaltzClientConfig(String configPrefix, Map<Object, Object> configValues)
     public SSLConfig getSSLConfig() {
         return new SSLConfig(configPrefix + CLIENT_SSL_CONFIG_PREFIX, configValues);
     }
+
+    /**
+     * Checks if code is running from a unit test
+     * @return true if yes, false otherwise
+     */
+    public static boolean isJUnitTest() {
+        for (StackTraceElement element : Thread.currentThread().getStackTrace()) {
+            if (element.getClassName().startsWith("org.junit.")) {
+                return true;
+            }
+        }
+        return false;
+    }
 }
+
```

Comment:
nit: In IntelliJ, it's showing me 2 new lines at the end of the file with this fix. Check it out once and fix it if its duplicate.

Reply:
fixed!